models/comments.model.js

```diff
@@ -0,0 +1,17 @@
+const db = require("../db/connection")
+const { fetchArticles } = require("./articles.models")
+
+exports.fetchCommentsByArticleId = (article_id) => {
+    let queryStr = "SELECT comments.comment_id, votes, created_at, author, body, article_id FROM comments";
+    queryStr += " WHERE comments.article_id = $1";
+    queryStr += " ORDER BY created_at DESC";
```

Comment:
I don't understand why you've constructed this string over three commands. Perhaps you wanted it to appear over multiple lines? In which case you can use backticks ` instead of speech marks ".

Reply:
Hi Jo, thanks for your feedback! I actually did this in anticipation of having to split the query string up later on to query the tables, or maybe it was because I thought I would have to allow customised sorting at this point...
I used the name queryVals because it was relating to values to be inserted into an SQL query string, but I see what you mean.